package.json

```diff
@@ -1,25 +1,29 @@
 {
-    "name": "storybook-builder-vite-parent",
-    "version": "0.1.13",
-    "description": "An experimental plugin to run and build Storybooks with Vite",
-    "main": "index.js",
-    "author": "",
-    "license": "MIT",
-    "scripts": {
-        "lint": "prettier --check .",
-        "format": "prettier --write ."
-    },
-    "workspaces": [
-        "packages/example-react",
-        "packages/example-svelte",
-        "packages/example-vue",
-        "packages/example-workspaces",
-        "packages/storybook-builder-vite"
-    ],
-    "prettier": {
-        "singleQuote": true
-    },
-    "devDependencies": {
-        "prettier": "^2.3.2"
-    }
+  "name": "storybook-builder-vite-parent",
+  "version": "0.1.13",
+  "description": "An experimental plugin to run and build Storybooks with Vite",
+  "license": "MIT",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "scripts": {
+    "lint": "cross-env NODE_ENV=production eslint \"packages/*/**/*.{js,jsx,mjs,cjs}\" --fix",
+    "lint-ci": "cross-env NODE_ENV=production eslint \"packages/*/**/*.{js,jsx,mjs,cjs}\""
```

Comment:
We shouldn't need to use `cross-env` here since we are using yarn berry.

Reply:
Part of the plan https://github.com/eirslett/storybook-builder-vite/issues/188 is:

> Workspaces: use NPM or Pnpm, because Yarn is not working out of the box like it should be.

More info about it in https://github.com/eirslett/storybook-builder-vite/issues/188#issuecomment-1004221826.

So, I used a cross-platform solution for scripts that works independently of the package manager.

Reply:
Maybe we can wait until that PR before making the change, if it's needed there?

Reply:
> Maybe we can wait until that PR before making the change, if it's needed there?

Yes, I can remove the `cross-env`.

Reply:
Fixed in c8e135c.